index.js

```diff
@@ -1,10 +1,21 @@
+const assert = require('assert')
 const fs = require('fs')
 const request = require('request-promise-native')
 
+const DEFAULT_POLL_INTERVAL = 5000
+
+const wait = (time = 5) => {
```

Comment:
Does it makes sense to default `time` to `DEFAULT_POLL_INTERVAL` here?

Reply:
Yes, that probably does make sense here.

Reply:
Ah, I realize now why I did that, it was to allow testing the polling option without long waits. I'll add a comment.

Reply:
Can't you pass that through `options.pollInterval` in the test?